README.md

```diff
@@ -293,6 +293,17 @@ over manual test execution. To view these options run:
 	man ctest
 
 
+### Building Ceph using Containers
+
+Ceph now provides tools to build the code, run unit tests, or build packages
+from within an OCI-style container using podman or docker! This allows one to
+build code for distributions other than the one you have on your system, avoids
+the need to install build dependecies for Ceph on your local system and
+provides an opportunity to test builds on platforms that are not yet supported
+by the offcial build infrastructure. For more details see the [container build
```

Comment:
official

Reply:
fixed.
/me needs to enable spellcheck when writing docs.